contracts/asset-proxy/contracts/src/bridges/CurveBridge.sol

```diff
@@ -26,22 +26,21 @@ import "@0x/contracts-exchange-libs/contracts/src/IWallet.sol";
 import "@0x/contracts-utils/contracts/src/DeploymentConstants.sol";
 import "../interfaces/IERC20Bridge.sol";
 import "../interfaces/ICurve.sol";
-import "./MixinGasToken.sol";
 
 
 // solhint-disable not-rely-on-time
 // solhint-disable space-after-comma
 contract CurveBridge is
     IERC20Bridge,
     IWallet,
-    DeploymentConstants,
-    MixinGasToken
+    DeploymentConstants
 {
     struct CurveBridgeData {
         address curveAddress;
+        bytes4 exchangeFunctionSelector;
+        address fromTokenAddress;
```

Comment:
Easier than figuring out whether to call `underlying_coins()` or just `coins()`.